a2ml/api/a2ml_model.py

```diff
@@ -70,50 +70,78 @@ def predict(self, filename,
             model_id(str): The deployed model id you want to use.
             threshold(float): For classification models only. This will return class probabilities with response.
             locally(bool): Predicts using a local model if True, on the Provider Cloud if False.
+            data: dict or array of records
+            columns(list): list of column names if data is array of records            
             output(str): Output csv file path.
             provider (str): The automl provider you wish to run. For example 'auger'. The default is None - use provider set in costructor or config.
 
         Returns:
             Results for provider. ::
+                if filename is not None:
 
                 {
                     'auger': {
                         'result': True,
-                        'data': {'model_id': 'D881079E1ED14FB'}
+                        'data': {'predicted': 'dataset_predicted.csv'}
                     }
                 }
+ 
+                if filename is None and data is not None and columns is None:
 
+                {
+                    'auger': {
+                        'result': True,
+                        'data': {'predicted': [{col1: value1, col2: value2}, {col1: value3, col2: value4}]}
+                    }
+                }
+
+                if filename is None and data is not None and columns is not None:
+
+                {
+                    'auger': {
+                        'result': True,
+                        'data': {'predicted': {'columns': ['col1', 'col2'], 'data': [['value1', 'value2'], ['value3', 'value4']]}}
+                    }
+                }
```

Comment:
This code block needs to be properly formatted, it currently renders like this:

<img width="1174" alt="Screen Shot 2020-06-29 at 1 52 56 PM" src="https://user-images.githubusercontent.com/30328334/86054904-db047680-ba0f-11ea-94c2-a2a1ff4d715b.png">